cmd/juju/metricsdebug/metrics.go

```diff
@@ -162,10 +168,19 @@ func formatTabular(writer io.Writer, value interface{}) error {
 	for _, col := range []int{1, 2, 3, 4} {
 		table.RightAlign(col)
 	}
-	table.AddRow("UNIT", "TIMESTAMP", "METRIC", "VALUE")
+	table.AddRow("UNIT", "TIMESTAMP", "METRIC", "VALUE", "LABELS")
 	for _, m := range metrics {
-		table.AddRow(m.Unit, m.Timestamp.Format(time.RFC3339), m.Metric, m.Value)
+		table.AddRow(m.Unit, m.Timestamp.Format(time.RFC3339), m.Metric, m.Value, renderLabels(m.Labels))
 	}
 	_, err := fmt.Fprint(writer, table.String())
 	return errors.Trace(err)
 }
+
+func renderLabels(m map[string]string) string {
+	var result []string
+	for k, v := range m {
+		result = append(result, fmt.Sprintf("%s=%s", k, v))
```

Comment:
Testing should ensure renderLabels and keyLabels and Labelskey produce the same output, so it's not confusing for user / us. I *think* you've covered this.